stdlib/3/lib2to3/pgen2/grammar.pyi

```diff
@@ -0,0 +1,26 @@
+# Stubs for lib2to3.pgen2.grammar (Python 3.6)
+
+from typing import Any, Dict, List, Optional, Tuple, TypeVar
+
+_P = TypeVar('_P')
+_Label = Tuple[int, Optional[str]]
+_States = List[List[Tuple[int, int]]]
+
+class Grammar:
+    symbol2number: Dict[str, int]
+    number2symbol: Dict[int, str]
+    states: _States
```

Comment:
This actually needs one more level of list. It's three nested lists, then the tuple. I think you should rename the type alias to "_DFA", then make this a `List[_DFA]`; that would match the comments in the code.